### PR TITLE
SW-2279 Nurseries Table

### PR DESCRIPTION
**Pull request by charlie-tf**
image
image

### CODE REVIEW

File: src/components/Nurseries/index.tsx [hunk 1]
```diff
@@ -95,22 +98,41 @@ export default function NurseriesList({ organization }: NurseriesListProps): JSX
...
-          </Typography>
+      <PageHeaderWrapper nextElement={contentRef.current}>
+        <Grid container paddingBottom={theme.spacing(4)} paddingLeft={isMobile ? 0 : theme.spacing(3)}>
+          <Grid item xs={9} alignItems='center'>
+            <Typography fontSize='24px' fontWeight={600}>
+              {strings.NURSERIES}
+            </Typography>
+          </Grid>
+          <Grid item xs={3} sx={{ textAlign: 'right' }}>
+            {['Admin', 'Owner'].includes(organization.role) &&
```
Comment (karthikbtf) [inline src/components/Nurseries/index.tsx:109]: we could use isAdmin from src/utils/organization (captures admin and owner). unrelated to your change for sure.

**Review by karthikbtf — APPROVED** [review]
Looks good to me 👍

**Comment by hannah-cai** [comment]
looks good! not sure if this is already being worked on in the other branch, but the icon size is also off here, and the border stroke is a bit cut off
image

**Comment by charlie-tf** [comment]
added margin to keep the searchBar border from being cut off. the icon size is addressed in the other PR and will carry over to here once this is merged.

**Pull request merged**